README.md

```diff
@@ -1,102 +1,105 @@
-# certbot-dns-desec
+# certbot-dns-desec: Get Let's Encrypt Certificates for Domains Hosted at deSEC
 
 ![main branch CI test status](https://github.com/desec-io/certbot-dns-desec/workflows/Tests/badge.svg?branch=main)
 [![pypi badge](https://img.shields.io/pypi/v/certbot-dns-desec.svg)](https://pypi.org/project/certbot-dns-desec/)
 
-⚠ This plugin is under development, API and CLI might change! ⚠
+Certbot plugin to obtain TLS certificates from Let's Encrypt for domains hosted with deSEC.io, using the DNS challenge
+challenge mechanism.
 
-[deSEC](https://desec.io/) DNS Authenticator plugin for [Certbot](https://certbot.eff.org/)
 
-This plugin automates the process of completing a ``dns-01`` challenge by
-creating, and subsequently removing, TXT records using the deSEC DNS API.
+## Installation
 
-## Configuration of deSEC
+To get certificates from Let's Encrypt, install certbot and this plugin.
+There are many ways to install certbot, this guide uses Python's `pip`:
 
-Log in at [deSEC](https://desec.io/) and create the domain you would like to use and a token for DNS management.
+```
+python3 -m pip install certbot certbot-dns-desec
+```
 
-## Installation
+## Prerequisites
 
-This certbot plugin can be installed using `pip`:
+To get a Let's Encrypt certificate for your domain,
+you need an access token for the deSEC.io account that holds the domain you need the certificate for.
```

Comment:
`you need a deSEC API token with sufficient permission for performing the required DNS changes on your domain.`